frontend/src/components/AlterSchemaPrepForm/GhostDialog.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <BBDialog
+    ref="dialog"
+    :title="$t('issue.migration-mode.title')"
+    :positive-text="$t('common.next')"
+  >
+    <div class="w-[24rem] space-y-4 pl-8 pr-2 pb-4">
+      <div class="flex items-start space-x-2">
+        <input
+          v-model="state.mode"
+          tabindex="-1"
+          type="radio"
+          class="text-accent disabled:text-accent-disabled focus:ring-accent mt-0.5"
+          value="normal"
+        />
+        <div @click="state.mode = 'normal'">
+          <div class="textlabel">
+            {{ $t("issue.migration-mode.normal.title") }}
+          </div>
+          <div class="textinfolabel mt-1">
+            <i18n-t tag="p" keypath="issue.migration-mode.normal.description" />
+          </div>
+        </div>
+      </div>
+      <div class="flex items-start space-x-2">
+        <input
+          v-model="state.mode"
+          tabindex="-1"
+          type="radio"
+          class="text-accent disabled:text-accent-disabled focus:ring-accent mt-0.5"
+          value="online"
+        />
+        <div @click="state.mode = 'online'">
+          <div class="textlabel">
+            {{ $t("issue.migration-mode.online.title") }}
+          </div>
+          <div class="textinfolabel mt-1">
+            <i18n-t tag="p" keypath="issue.migration-mode.online.description">
+              <template #link>
+                <LearnMoreLink
+                  url="https://github.com/bytebase/bytebase/blob/main/docs/design/gh-ost-integration.md"
+                />
+              </template>
+            </i18n-t>
+          </div>
+        </div>
+      </div>
+    </div>
+  </BBDialog>
+</template>
+
+<script lang="ts" setup>
+import { reactive, ref } from "vue";
+import { BBDialog } from "@/bbkit";
+import LearnMoreLink from "../LearnMoreLink.vue";
+
+type Mode = "normal" | "online";
+
+const dialog = ref<InstanceType<typeof BBDialog> | null>(null);
+
+const state = reactive({
+  mode: "normal" as Mode,
```

Comment:
We can declare the `LocalState` type firstly, so that we don't need to use `as Mode`.

Reply:
Updated, PTAL